### PR TITLE
cmd/gentypes: generate response and event ctor

### PR DESCRIPTION
**Pull request by suzmue**
The mapping from event / response id to event and response types
can be done in generating. This makes sure that the maps stay up
to date with the spec.

### CODE REVIEW

**Review by hyangah — APPROVED** [review]
LGTM

File: schematypes.go [hunk 1]
```diff
@@ -1901,3 +1901,126 @@ type DisassembledInstruction struct {
...
+	"initialize": func() Message {
+		return &InitializeRequest{
+			Arguments: InitializeRequestArguments{
+				// Set the default values specified here: https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Initialize.
+				LinesStartAt1:   true,
+				ColumnsStartAt1: true,
+				PathFormat:      "path",
+			}}
+	},
+	"configurationDone":         func() Message { return &ConfigurationDoneRequest{} },
```
Comment (polinasok) [inline schematypes.go:1919]: Not critical: but how is the spacing lined up here, but not above?

**Review by polinasok — APPROVED** [review]
Oh, I completely forgot we were not generating these! Thank you for moving this to the generator!

File: schematypes.go (hunk 1 above)
Comment (suzmue) [reply schematypes.go:1973]: seems like this is just how the go formatter works!

**Pull request merged**